apps/expo/src/app/index.tsx

```diff
@@ -25,20 +26,104 @@ export default function Home() {
     return <Redirect href="/sign-in" />;
   }
 
+  const getInitials = (name: string) => {
+    const initials = name
+      .split(" ")
+      .map((word) => word[0] + (word[1] || ""))
+      .join("")
+      .toUpperCase();
+    return initials;
+  };
+
+  const userName = user?.username!;
+  const userInitials = getInitials(userName);
+
   return (
-    <ScreenView>
-      <Text>Home</Text>
-      <Button
-        variant="destructive"
-        onPress={async () => {
-          setLoading(true);
-          await clerk.signOut();
-          setLoading(false);
-        }}
-        loading={loading}
-      >
-        <Text>Sign out</Text>
-      </Button>
-    </ScreenView>
+    <View className="flex-1 bg-white p-6">
+      <View className="mb-4 flex-row items-center justify-between">
+        <View className="mt-4 flex-row items-center">
+          <View className="h-10 w-10 items-center justify-center rounded-full bg-gray-300">
+            <Text>{userInitials}</Text>
+          </View>
+          <View className="ml-2">
+            <Text className="text-lg text-black">Olá, {userName}</Text>
+            <Text className="mt-1 text-sm text-gray-600">
+              Bem-vindo de novo!
+            </Text>
+          </View>
+        </View>
+        <View className="mb-4 flex-row items-center justify-end">
```

Comment:
`flex-row` ele seta a propriedade `flex-direction` do css, para realmente se tornar um display flex precisa da classe `flex`,  e é bom evitar tanto uso de margin, pq fica meio estranho se a gente for testar em tamanhos de telas diferenciados